File: 1º Semestre/Programação de Computadores/Lista de Exercicios 1/14.cpp
#include <iostream>
#include <string>
using namespace std;

int main(){
	float numero1, numero2;
	
	printf("Digite o primeiro numero real: ");
	scanf("%f", &numero1);
	
	printf("Digite o segundo numero real: ");
	scanf("%f", &numero2);
	
	printf("O produto dos numeros %0.02f e %0.02f corresponde a %0.02f", numero1, numero2, numero1 * numero2);
}

File: 1º Semestre/Programação de Computadores/Lista de Exercicios 1/Python/14.py
"""
14.	Escreva um programa que solicite ao usuário dois números reais e ao final apresente na tela 
o produto dos dois números informados da seguinte forma: "O produto dos numeros N e X corresponde a Y" 
"""

floatNum1 = float(input("Digite o primeiro numero reais: "))
floatNum2 = float(input("Digite o segundo numero reais: "))

print("O produto dos numeros {} e {} correspondem a {}".format(floatNum1, floatNum2, floatNum1 * floatNum2))